projects/js-packages/publicize-components/src/components/share-status/styles.module.scss

```diff
@@ -77,6 +77,7 @@
 	// Use :global scope to avoid CSS encapsulation
 	:global {
 		// Use import without ".css" extension to avoid it being rendered as URL
+		/* stylelint-disable-next-line no-invalid-position-at-import-rule -- this import is intentionally scoped */
```

Comment:
I wonder if we should just disable this rule for scss files, with something like
```js
    overrides: [
        {
            files: [ '*.scss', '**/*.scss' ],
            rules: {
                // Sass's @import is not CSS @import.
                'no-invalid-position-at-import-rule': null,
            },
        },
    ],
```

Reply:
I thought about that, but decided against it for two reasons:

1. I'd like to avoid as many "disables" as possible so we can stay close to the core rules.
2. `@import` ideally wouldn't be used in new code anymore; `@use` is the preferred SASS import, and it must come before style rules anyway ([docs](https://sass-lang.com/documentation/at-rules/use/)). Enforcing things at the top makes it easier to transition to `@use` down the road.